addons/mail/static/src/components/rtc_call_participant_card/rtc_call_participant_card.xml

```diff
@@ -47,7 +47,7 @@
                             </t>
                         </span>
                         <div class="o_RtcCallParticipantCard_overlayTop">
-                            <t t-if="callParticipantCard.rtcSession.isMuted">
+                            <t t-if="callParticipantCard.rtcSession.isMutedOrDeaf">
```

Comment:
we should only display one icon, deaf in priority, and mute icon only if not deaf